src/components/calcite-option/calcite-option.e2e.ts

```diff
@@ -0,0 +1,53 @@
+import { newE2EPage } from "@stencil/core/testing";
+import { accessible, defaults, reflects, renders } from "../../tests/commonTests";
+
+describe("calcite-option", () => {
+  it("renders", async () => renders("calcite-option"));
+
+  it("is accessible", async () => accessible("calcite-option"));
+
+  it("has defaults", async () =>
+    defaults("calcite-option", [
```

Comment:
future enhancement and unrelated, but had a thought the other day that this syntax could be a lot more terse, like instead of an array we could update it to just pass accept an object:

```
defaults("calcite-option", {
  "disabled": false
});
```